flexget/utils/template.py

```diff
@@ -187,6 +187,35 @@ def filter_get_year(name: str) -> str:
     return split_title_year(name).year
 
 
+def filter_parse_size(val: str, match_re: Optional[str] = None) -> int:
+    """Parse human-readable file size to bytes"""
+    if not isinstance(val, str):
+        return val
+
+    match_re = match_re or r'^(?P<digit>\d+(?:\.\d+)?)\s*(?P<unit>[a-zA-Z]+B)$'
+    matched_size = re.match(match_re, val)
+
+    if matched_size:
+        unit = matched_size['unit']
+        size_map = {
+            'B': 1,
+            'KB': 1024,
+            'KiB': 1024,
+            'MB': 1024**2,
```

Comment:
should this be `1000**2`?

Reply:
> should this be `1000**2`?

idk :( It may depends on platform that produce this string. 

In my opinion, it is providing an approximate value of torrent size because of the limited decimal digits, so 1024 or 1000 doesn't matter.